examples/datasource-streaming-backend-websocket/streaming-backend-websocket-plugin/src/datasource.ts

```diff
@@ -0,0 +1,42 @@
+import {
+  CoreApp,
+  DataQueryRequest,
+  DataQueryResponse,
+  DataSourceInstanceSettings,
+  LiveChannelScope,
+} from '@grafana/data';
+import { DataSourceWithBackend, getGrafanaLiveSrv } from '@grafana/runtime';
+
+import { defaults } from 'lodash';
+import { Observable, merge } from 'rxjs';
+
+import { MyQuery, MyDataSourceOptions, DEFAULT_QUERY } from './types';
+
+export class DataSource extends DataSourceWithBackend<MyQuery, MyDataSourceOptions> {
+  constructor(instanceSettings: DataSourceInstanceSettings<MyDataSourceOptions>) {
+    super(instanceSettings);
+  }
```

Comment:
nit: if you are not modifying anything you can skip this?

Reply:
I've notice this code is comes from `npx @grafana/create-plugin@latest`.
Should we also remove it from the template later?

Reply:
Actually this is necessary indeed. I've removed it and the datasource frontend broke